.circleci/config.yml

```diff
@@ -6,6 +6,44 @@ references:
   postgres: &postgres postgres:11.6
   db_migrate: &db_migrate ${AWS_ACCOUNT_ID}.dkr.ecr.us-west-2.amazonaws.com/easi-db-migrate:${CIRCLE_SHA1}
 jobs:
+  restart_impl_ecs_app_service:
+    docker:
+      - image: *circleci_docker_primary
+    steps:
+      - checkout
+      - run:
+          name: Setup custom environment variables
+          command: |
+            temp_role=$(aws sts assume-role \
+                    --role-arn arn:aws:iam::$AWS_ACCOUNT_ID_IMPL:role/circleci \
+                    --role-session-name circleci)
+            echo "export AWS_ACCESS_KEY_ID=$(jq --raw-output .Credentials.AccessKeyId <<< "$temp_role")" >> $BASH_ENV
+            echo "export AWS_SECRET_ACCESS_KEY=$(jq --raw-output .Credentials.SecretAccessKey <<< "$temp_role")" >> $BASH_ENV
+            echo "export AWS_SESSION_TOKEN=$(jq --raw-output .Credentials.SessionToken <<< "$temp_role")" >> $BASH_ENV
+      - run:
+          name: Restart ECS app service
+          command: |
+            ./scripts/deploy_service "easi-app" ""
```

Comment:
Is the empty string necessary or will the deploy_service script default to an empty value for image if no second parameter is provided?

Reply:
`""` is the default value of any empty argument. I debated which option is least confusing and settled on this because it might be more clear that omitting the argument is intentional. I don't feel strongly about this though since it seems awkward at any rate.

Reply:
I don't feel strongly about it either, but I agree it seems a little awkward.